### PR TITLE
Jwa review of lesson4

### PR DESCRIPTION
**Pull request by tir38**
adds comments for lesson 4

### CODE REVIEW

File: src/LessonFour_AdvancedStreams.java [hunk 1]
```diff
@@ -4,52 +4,90 @@
...
+    Now its worth nothing this is different from what we did when we nested Observables. In that case we always had one stream.
+    Lets take a stream of integers and a stream of strings and join them.
+    */
     @Test
     public void merging() {
         Observable<Object> you = Observable.just(1, 2, 3);
         Observable<String> me = Observable.just("A", "B", "C");
-        you.mergeWith(me).subscribe(string -> mReceived += string + " ");
+        you.mergeWith(me)
+                .subscribe(string -> mReceived += string + " ");
```
Comment (joshskeen) [inline src/LessonFour_AdvancedStreams.java:33]: :+1:

**Comment by joshskeen** [comment]
:boom:

**Comment by tir38** [comment]
Is that a good :boom: or a bad :boom: ?

**Comment by joshskeen** [comment]
it's a Great :boom:

**Pull request merged**